### PR TITLE
Add MobileNetV1 and MobileNetV1 test into ModelZoo 

### PR DESCRIPTION
**Pull request by warthecatalyst**
Add MobileNet version1 (reference paper https://arxiv.org/pdf/1704.04861.pdf) into ModelZoo and create a model test to check out whether the shapes and values of parameters meet our expectations

### CODE REVIEW

File: integration/src/main/java/ai/djl/integration/tests/model_zoo/classification/MobileNetV1Test.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
+package ai.djl.integration.tests.model_zoo.classification;
```
Comment (frankfliu) [inline integration/src/main/java/ai/djl/integration/tests/model_zoo/classification/MobileNetV1Test.java:1]: Please add license header to this java file

File: model-zoo/src/main/java/ai/djl/basicmodelzoo/cv/classification/MobileNetV1.java [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+ * {@code MobileNetV1} contains a generic implementation of Mobilenetw adapted from
+ * https://github.com/weiaicunzai/pytorch-cifar100/blob/master/models/mobilenet.py (Original author weiaicunzai)
+ *
+ * see https://arxiv.org/pdf/1704.04861.pdf for more information about MobileNet
+ */
+public final class MobileNetV1 {
+    private MobileNetV1(){}
+
+    /**
+     * Builds a {@link Block} that represent a depthWise-pointWise Unit used in the implementation of
```
Comment (frankfliu) [inline model-zoo/src/main/java/ai/djl/basicmodelzoo/cv/classification/MobileNetV1.java:43]: First javadoc sentence should end with a period.

File: model-zoo/src/main/java/ai/djl/basicmodelzoo/cv/classification/MobileNetV1.java [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+     * the MobileNet Model
+     *
+     * @param inputChannels number of inputChannels, used for depthWise Kernel
+     * @param outputChannels number of outputChannels, used for pointWise kernel
+     * @param stride control the stride of depthWise Kernel
+     * @param builder add the builder to obtain batchNormMomentum
+     * @return a {@link Block} that represent a depthWise-pointWise Unit
+     */
+    public static Block DepthSeparableConv2d(int inputChannels,int outputChannels,int stride,Builder builder){
```
Comment (frankfliu) [inline model-zoo/src/main/java/ai/djl/basicmodelzoo/cv/classification/MobileNetV1.java:52]: java method name should start with lower case

File: model-zoo/src/main/java/ai/djl/basicmodelzoo/cv/classification/MobileNetV1.java [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+                                .optEpsilon(2E-5f)
+                                .optMomentum(builder.batchNormMomentum)
+                                .build())
+                .add(Activation.reluBlock());
+
+        return depthWise.add(pointWise);    //two blocks are merged together
+    }
+
+    /**
+     * Create a new {@link Block} of {@link MobileNetV1} with the arguments from the given {@link
```
Comment (frankfliu) [inline model-zoo/src/main/java/ai/djl/basicmodelzoo/cv/classification/MobileNetV1.java:88]:
```suggestion
     * Creates a new {@link Block} of {@link MobileNetV1} with the arguments from the given {@link
```

File: model-zoo/src/main/java/ai/djl/basicmodelzoo/cv/classification/MobileNetV1.java [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+        float alpha = 1f;   //width multiplier defined in the paper
+
+        long outSize = 10;  //10 as default for basic datasets like cifar10 or mnist
+
+        private final int[] filters = new int[]{32,64,128,128,256,256,512,512,1024,1024};
+
+        Builder(){}
+
+        /**
+         * Set the widthMultiplier of MobileNet
```
Comment (frankfliu) [inline model-zoo/src/main/java/ai/djl/basicmodelzoo/cv/classification/MobileNetV1.java:180]:
```suggestion
         * Sets the widthMultiplier of MobileNet.
```

**Review by frankfliu — COMMENTED** [review]
Thanks for your contribution. A few general comment:
1. Please run `./gradlew fJ` to reformat your java code before you commit the code locally.
2. Please run `./gradlew check -x test` to make sure all static code analysis can pass
3. Please run `./gradlew build` to run all unitest

**Comment by warthecatalyst** [comment]
thanks and I will make my changes
















At 2022-07-20 12:30:52, "Frank Liu" ***@***.***> wrote:

@frankfliu commented on this pull request.

In model-zoo/src/main/java/ai/djl/basicmodelzoo/cv/classification/MobileNetV1.java:

> +
+    /**
+     * The Builder to construct a ***@***.*** MobileNetV1} object.
+     */
+    public static final class Builder{
+        float batchNormMomentum = 0.9f;
+        float alpha = 1f;   //width multiplier defined in the paper
+
+        long outSize = 10;  //10 as default for basic datasets like cifar10 or mnist
+
+        private final int[] filters = new int[]{32,64,128,128,256,256,512,512,1024,1024};
+
+        Builder(){}
+
+        /**
+         * Set the widthMultiplier of MobileNet


A few more suggestion:

Please push your code into a separate branch (other than master branch), so we can make changes on your branch to collaborate with your PR
Please rebase your branch with latest master branch to make it easy to review your changes.

**Pull request closed without merging**